CONTRIBUTING.md

```diff
@@ -123,7 +122,7 @@ While the prerequisites above must be satisfied prior to having your pull reques
 
 ### 6. Translating guide
 
-We are happy to have people volunteer to translate the App. Just follow the steps as described [in our wiki](https://github.com/purefinance/pure.finance/wiki).
+We are happy to have people volunteer to translate the App. Just follow the steps as described [in our wiki](https://github.com/hemilabs/pure.finance/wiki).
```

Comment:
there's no wiki (it seems it is not copied when forking), so we should either drop this or manually copying the contents of the PF wiki

Reply:
Or we may keep the link to the original wiki.